Public Record Office Victoria.js

```diff
@@ -0,0 +1,216 @@
+{
+	"translatorID": "99592877-f698-4e14-b541-b6181f6c577f",
+	"label": "Public Record Office Victoria",
+	"creator": "Tim Sherratt (tim@timsherratt.au)",
+	"target": "^https?://prov\\.vic\\.gov\\.au/(archive|search_journey)/*",
+	"minVersion": "5.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2024-01-21 03:13:11"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2024 Tim Sherratt
+
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+async function detectWeb(doc, url) {
+	if (getSearchResults(doc, true)) {
+		return "multiple";
+	// match items but not series, agencies, or functions (VPRS, VA or VF)
+	}
+	else if (url.match(/archive\/(?!VPRS|VA|VF)[A-Z0-9-]+/)) {
+		return "manuscript";
+	}
+	return false;
+}
+
+function getSearchResults(doc, checkOnly) {
+	let items = {};
+	let found = false;
+	let rows = doc.querySelectorAll(".sub_heading_search_result");
+	for (let row of rows) {
+		let href = row.href;
+		href = href.match(/archive\/(?!VPRS|VA|VF)[A-Z0-9-]+/) ? href : null;
+		let title = row.innerText.replace(/\n/g, " ");
+		if (!href || !title) continue;
+		if (checkOnly) return true;
+		found = true;
+		items[href] = title;
+	}
+	return found ? items : false;
+}
+
+async function doWeb(doc, url) {
+	if (await detectWeb(doc, url) == 'multiple') {
+		let items = await Zotero.selectItems(getSearchResults(doc, false));
+		if (items) {
+			for (let url of Object.keys(items)) {
+				await scrape(url);
+			}
+		}
+	}
+	else {
+		await scrape(url);
+	}
+}
+
+async function scrape(url) {
+	// Get the API record for this item
+	let id = url.match(/archive\/([A-Z0-9-]+)/)[1];
+	let apiURL = "https://api.prov.vic.gov.au/search/query?wt=json&q=(_id%3A" + id + ")";
+	let apiJSON = await requestJSON(apiURL);
+	let record = apiJSON.response.docs[0];
+	
+	// Create the Zotero item
+	let item = new Zotero.Item("manuscript");
+	item.title = record.title;
+	item.type = record.category;
+	item.archive = "Public Record Office Victoria";
+	item.archiveLocation = record.citation;
+	item.libraryCatalog = "PROV API";
```

Comment:
I think something more readable -- like "Public Record Office Victoria" is perferable here.

Reply:
Yep, changed to 'Public Record Office Victoria API'.